Engine/Graphics/OpenGL/RenderOpenGL.cpp

```diff
@@ -105,11 +105,11 @@ bool IsBModelVisible(BSPModel *model, int reachable_depth, bool *reachable) {
     }
 
     // view range check
-    if (dist <= pIndoorCameraD3D->GetFarClip() + 2048) {
+    if (dist <= pCamera3D->GetFarClip() + 2048) {
         // boudning point inside cone
         if (sloperem >= 0) return true;
         // bounding radius inside cone
-        if (abs(sloperem) < model->sBoundingRadius + 512) return true;
+        if (abs(sloperem) < model->sBoundingRadius /*+ 512*/) return true;
```

Comment:
Was + 512 added before as a hack?

Reply:
Will revert. This needs to be in, it was commented during testing. The additional slack stops buildings dissapearing at extents of camera pitch.